sormas-ui/src/main/java/de/symeda/sormas/ui/UiUtil.java

```diff
@@ -19,14 +21,26 @@ public static boolean permitted(Set<FeatureType> features, UserRight userRight)
 		return enabled(features) && permitted(userRight);
 	}
 
+	public static boolean permitted(Set<FeatureType> features, Set<UserRight> userRights) {
```

Comment:
this method looks to be always used with a single feature type, so why require a set of feature types here?
also why not use vararg for userRights, would be easier to use and wouldn't need to transform the set into an array in the `permitted` method

Reply:
Yes, I've assumed that after replacing all the occurences of FacadeProvider.getFeatureConfigurationFacade().isFeatureEnabled(featureType) in the project there will be places where multiple featureTypes will be necessary, but I've checked now the code and it looks like right now always just one featureType is used, as you've said. 

I also changed the userRights to use varargs.